app/views/mcda/prompt.py

```
@@ -98,17 +100,44 @@ def get_axis_description(axis_data: dict) -> str:
+        |---|---|
+        |admin_national|one value per country (ISO 3166-1 boundary)|
+        |admin_subnational|any aggregated admin unit below the country level (admin-1, admin-2, counties, districts…)|
+        |grid_coarse|regularly-spaced rasters or H3 > ≈1 km (250 m GHS-POP counts as coarse once aggregated)|
+        |grid_fine|rasters or H3 ≤ ≈1 km (250 m, 100 m, 30 m, etc.)|
+        |feature_derived|counts, densities or proximities based on discrete features (points, lines, polygons)|
+    '''
+
+    layer_temporal_ext_table = '''
+        |value|meaning|
+        |---|---|
+        |static|fixed value; doesn’t move with the calendar|
```

Ruff (0.12.2)

String contains ambiguous `’` (RIGHT SINGLE QUOTATION MARK). Did you mean ``` (GRAVE ACCENT)?

(RUF001)